index.html

```diff
@@ -152,18 +152,18 @@
         <section id="aparatus" class="main__aparatus aparatus">
           <div class="aparatus__container container">
             
-                <h2 class="aparatus__Title Title">АВТОМАТИЗОВАНИЙ ПРОГРАМНО-АПАРАТНИЙ КОМПЛЕКС «ГІПОТРОН-М»</h2> 
+                <h2 class="aparatus__title title">АВТОМАТИЗОВАНИЙ ПРОГРАМНО-АПАРАТНИЙ КОМПЛЕКС «ГІПОТРОН-М»</h2> 
                  
-                <div class="aparatus__coplex coplex">
-                  <div> <img src="img/Aparatus.png" alt="Aparatus" class="center"></div>
-                  <div class="aparatus__backInfoAparatus backInfoAparatus">
-                    <div class="aparatus__infoAparatus infoAparatus">ГІПОТРОН-М - унікальна установка, 
+                <div class="aparatus__about about-aparatus">
+                  <div> <img src="img/Aparatus.png" alt="Aparatus" class="about-aparatus__image"></div>
+                  <div class="about-aparatus__infoAparatus infoAparatus">
+                    <div class="infoAparatus__text text">ГІПОТРОН-М - унікальна установка, 
                       основана на індивідуальній регуляції дозованої нормобаричної гіпоксії, 
                       створеної методом зворотного дихання.</div>
                   </div>
```

Comment:
It would be better if you remove div with the class "about-aparatus__infoAparatus" and use only one div with the class named "about-aparatus__text"